Source/Views/AlertView.swift

```diff
@@ -72,6 +73,15 @@ class AlertView: AlertControllerView {
         }
     }
 
+    private func createBorders() {
+        if let borderColor = self.visualStyle.borderColor {
+            self.layer.borderColor = borderColor.cgColor
+        }
+        if let borderWidth = self.visualStyle.borderWidth {
+            self.layer.borderWidth = borderWidth
+        }
+    }
```

Comment:
Uh, sorry. It took a while to get back to you.

Are you sure about the `borderColor`?

It says on [CALayer.borderColor docs](https://developer.apple.com/documentation/quartzcore/calayer/1410903-bordercolor) that _"The default value of this property is an opaque black color."_. Does setting that to `nil` set the color to the _opaque black color_ or does it do something else? I'm not sure.

The default value of `borderWidth` seems to be 0.0 so that I'll accept without any objections. ;)